build.cake

```diff
@@ -38,17 +38,14 @@ Task("Test")
     });
 });
 
+// windows only because both inspectcore and nvika depend on net45
 Task("InspectCode")
+.WithCriteria(IsRunningOnWindows())
 .IsDependentOn("Compile")
 .Does(() => {
     var nVikaToolPath = GetFiles("./tools/NVika.MSBuild.*/tools/NVika.exe").First();
     var nugetToolPath = GetFiles("./tools/NuGet.CommandLine.*/tools/NuGet.exe").First();
```

Comment:
`dotnet build` automatically does a nuget restore. this is unnecessary.

Reply:
The tests are not restored at that point of time (this runs before the tests). Windows nuget makes no sense here, yes.